hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/HttpFSAuthenticationFilter.java

```diff
@@ -69,27 +72,11 @@
   @Override
   protected Properties getConfiguration(String configPrefix,
       FilterConfig filterConfig) throws ServletException{
-    Properties props = new Properties();
+    System.out.println("getConfiguration1");
```

Comment:
Please get rid of all System.out.println calls or alternatively, use the Logger.info method.

Reply:
Done.